python/whylogs/core/resolvers.py

```diff
@@ -67,6 +110,14 @@ def resolve(self, name: str, why_type: DataType, column_schema: ColumnSchema) ->
         result: Dict[str, Metric] = {}
         for m in metrics:
             result[m.name] = m.zero(column_schema.cfg)
+
+        if self.additional_metrics:
+            additional_metrics_result = self.resolve_additional_standard_metrics(
+                result=result, name=name, why_type=why_type, column_schema=column_schema
+            )
+            for key in additional_metrics_result.keys():
+                result[key] = additional_metrics_result[key]
+
         return result
 
 
```

Comment:
Consider adding an EmptyResolver? I do like the HistogramCountingTrackingResolver as a useful base to add onto but it might be better to allow the caller to skip most columns for a very lightweight profile.

Reply:
`DeclarativeSchema` is also a handy way to construct a lightweight profile :)